default/stringtables/en.txt

```diff
@@ -2851,10 +2851,16 @@ MAP_PROD_WASTED_TITLE
 Unused Production
 
 # %1% number of production points generated.
-# %2% number of production points currently not used for production.
+# %2% number of production points currently neither used for production nor for stockpile.
+# %3% yield ratio of importing PP into the imperial stockpile (the other part goes to waste)
+# %4% number of production points transferred to the imperial stockpile.
+# %5% number of production points currently neither used for production nor for stockpile.
 MAP_PROD_WASTED_TEXT
 '''Total Production : %1%
-Wasted Production : %2%
+Overproduction : %2%
```

Comment:
Over -> Excess ?

Reply:
I'd think overproduction would be correct (technical?) term. I don't know about daily usage though.
wikipedia: In economics, overproduction, oversupply, excess of supply or glut refers to excess of supply over demand of products being offered to the market.
Also google search for overproduction results in 2M hits and "excess production" is 400k

Reply:
Overproduction is having too much of something produced, not having too much capacity to produce something. The latter is being described.

Reply:
So "Total Production" should be "Production Capacity" and "Overproduction" should be "Excess Production" (or maybe "Demand Deficit")?

Reply:
Or rather "Unused Capacity"?